lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+  attr_reader :cards, :count
+    def initialize(cards)
+      @cards = cards
+      @count = cards.length
+      end
+
+    def cards_in_category(category)
+      cat_arr = []
+
+      cards.each do |card|
+        if card.category == category
+        cat_arr << card
+      end
+    end
+
+    cat_arr
+    end
```

Comment:
Keep your eye on syntax. We always want to make sure our end's match up with their corresponding block starter (def, if, each, etc.)